editor/src/components/canvas/ui-jsx-canvas.spec.tsx

```diff
@@ -1222,9 +1228,14 @@ export var ${BakedInStoryboardVariableName} = (props) => {
             width: 400px;
             height: 400px;
           \\"
+          data-paths=\\":utopia-storyboard-uid\\"
           data-uid=\\"utopia-storyboard-uid\\"
         >
-          <div id=\\"nasty-div\\" data-uid=\\"2a7~~~1 150~~~2 2f5~~~1 aaa scene-aaa\\">
+          <div
+            id=\\"nasty-div\\"
+            data-uid=\\"2a7~~~1 150~~~2 2f5~~~1 aaa scene-aaa\\"
+            data-paths=\\":150~~~2/2a7~~~1 :150~~~2 :2f5~~~1 utopia-storyboard-uid/scene-aaa:aaa :utopia-storyboard-uid/scene-aaa\\"
```

Comment:
Again, I think since `Thing` isn't focused, the paths should end at `aaa`

Reply:
in this test case "App" is an arbitrary js. this is the last known case where the template path is going to be gibberish, but since the editor does not support editing these projects, this is not a regression, just making an existing problem explicit